index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <script defer src="script.js"></script>
+    <title>Survey Form</title>
+</head>
+<body>
+    
+    
+        
+    <header class="header">
+        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
+        <p id="description" class="description text-center">Thank you for taking the time to help us improve the platform</p>
+    </header>
+
+    <div class="container">
+        <form action="" id="survey-form">
+            <label id="name-label">Name</label><br>
+            <input type="text" id="name" name="name" placeholder="Enter your name" required><br>
+            <label id="email-label">Email</label><br>
+            <input type="email" id="email" name="email" placeholder="Enter your email" required><br>
+            <label id="number-label">Age (optional)</label><br>
+            <input type="number" id="number" name="age" min="10" max="99" placeholder="age"><br><br>
+    
+            <label for="">Which option best describes your current role?</label><br>
+            <select name="" id="dropdown">
+                <option value="">Select Current Role</option>
+                <option value="">Student</option>
+                <option value="">Full Time Job</option>
+                <option value="">Full Time Learner</option>
+                <option value="">Prefer Not To Say</option>
+                <option value="">Other</option>
+            </select><br><br>
+    
+            <label for="">Would you recommend freeCodeCamp to a friend?</label><br>
```

Comment:
Please I suggest that you take away the spacing, this will make your code clean and readable.  See the above examples as  earlier highlighted